content/docs/projects/trust-manager.md

```diff
@@ -56,79 +70,299 @@ spec:
+when installing a public trust bundle in a Debian container.
+
+trust-manager has been designed in such a way that any version of any default CA package should work with any
+version of trust-manager which supports default CAs (`v0.4.0` and later). There should be no risk to the stability
+of trust-manager from upgrading.
+
+If you're using an official cert-manager-provided Debian CA package (which is the default), you should check which version you have
+and compare against the [latest package version](https://quay.io/repository/jetstack/cert-manager-package-debian?tab=tags&tag=latest).
+
+The version can be configured with the `.defaultPackageImage.tag` value on the Helm chart, and the version
+is also written to the `status` field on any synced Bundle resource which uses the default CA package.
+
+## Upgrading A Default CA Package Using Helm
+
+Say we want to do an in-place upgrade of our default CA package to tagged version `XYZ` - without upgrading trust-manager.
+
+We'll assume the Helm release is called "trust-manager" and that we've installed into the `cert-manager` namespace.
+
+> ⚠️ This upgrade process assumes that it's the only thing running. If another user or process changes Helm values
+> while you're doing this process, you might overwrite their work.
+
+First, we'll dump our current Helm values, so we don't lose them:
+
 ```bash
-kubectl get cm -A --field-selector=metadata.name=example-bundle
-NAMESPACE            NAME             DATA   AGE
-cert-manager         example-bundle   1      2m18s
-default              example-bundle   1      2m18s
-kube-node-lease      example-bundle   1      2m18s
-kube-public          example-bundle   1      2m18s
-kube-system          example-bundle   1      2m18s
-local-path-storage   example-bundle   1      2m18s
+helm get values -n cert-manager trust-manager -oyaml > values.yaml
 ```
 
+Next, if `defaultPackageImage.tag` is already set in `values.yaml`, update it. Otherwise, add it.
+You can find the available tags [on `quay.io`](https://quay.io/repository/jetstack/cert-manager-package-debian?tab=tags&tag=latest).
+
+```yaml
+# values.yaml
+...
+defaultPackageImage:
+  tag: XYZ
+```
+
+Finally, apply back the changes, being sure to manually specify the version of trust-manager which is installed, to avoid
+also updating the trust-manager controller at the same as the default CA package:
+
 ```bash
-kubectl get cm -n kube-system example-bundle -o jsonpath="{.data['target-key']}"
-123
-ABC
-hello world!
+# Get the currently installed version. You could do this manually if you find that easier.
+TRUST_MANAGER_VER=$(helm list --filter "^trust-manager$" -n cert-manager -ojson | jq -r ".[0].app_version")
+
+# Check the version makes sense
+echo $TRUST_MANAGER_VER
+
+# Run the upgrade
+helm upgrade -f values.yaml -n cert-manager trust-manager jetstack/trust-manager --version $TRUST_MANAGER_VER
 ```
+
+## Avoiding Gotchas
```

Comment:
Can we make this a more important section, like "read this before starting" or "Read this before running in production"? To me "Gotchas" seems like something I won't read because I know what I'm doing.

Reply:
I've tried rewriting a little, what do you think?